data_steward/tools/load_submission.py

```diff
@@ -0,0 +1,124 @@
+"""
+Loads a submission into bq from the archive
+"""
+import logging
+import argparse
+
+from google.cloud.bigquery import LoadJobConfig, Table
+from google.cloud.storage import Client
+
+from common import AOU_REQUIRED, JINJA_ENV
+from utils import pipeline_logging
```

Comment:
:nit:  I would probably import this under `if __name__=='__main__':` so that it is only imported if run as the main script.